ADAL.NET.sln

```diff
@@ -57,9 +57,6 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "XFormsApp", "tests\TestApp\XFormsApp\XFormsApp.csproj", "{856EA096-2283-49B6-B2CD-5C603551D150}"
 EndProject
 Project("{FF286327-C783-4F7A-AB73-9BCBAD0D4460}") = "ADAL.NuGet", "src\ADAL.NuGet\ADAL.NuGet.nuproj", "{E688082D-84FA-4B7C-B05B-AFDB2EFADB3F}"
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AdalMacTestApp", "tests\TestApp\AdalMacTestApp\AdalMacTestApp.csproj", "{5FC8BC8E-01B5-4C1C-9F60-D6402D793BB6}"
```

Comment:
Will have to put these two back, it loos like the change not to remove the mac bits was not staged. Will fix that